openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -44,7 +44,7 @@ class Meta(object):
         app_label = 'course_overviews'
 
     # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
-    VERSION = 7
+    VERSION = 6
```

Comment:
@nasthagiri if things in the stage DB are at v7 will going back to 6 cause any issues?

Reply:
There are several courses at version 7 now:
```
mysql> select version, count(version) from course_overviews_courseoverview group by version;
+---------+----------------+
| version | count(version) |
+---------+----------------+
|       4 |              7 |
|       6 |           1444 |
|       7 |            154 |
+---------+----------------+
3 rows in set (0.00 sec)
```

They will continue to remain at version 7 even after the revert.  When we push forward again, we'll need to force update them (which will need to be part of the new process).

Reply:
when you say force update them, do you mean as a part of how the code works? or as a run of the management command?

Reply:
well, we still need to figure out next steps on this.  one possibility includes (1) introducing a read-version and a write-version, (2) updating the write-version, (3) running a management command (with force-update), (4) updating the read-version.

Reply:
@nasthagiri long term plans aside, does anything need to happen in stage before or after this PR gets released to prevent issues with courseoverview versions?

Reply:
nothing - other than usual django migrations.